gallery/src/styles/NxCard/NxCardPage.tsx

```diff
@@ -0,0 +1,152 @@
+        <thead>
+          <tr className="nx-table-row nx-table-row--header">
+            <th className="nx-cell nx-cell--header">Class</th>
+            <th className="nx-cell nx-cell--header">Location</th>
+            <th className="nx-cell nx-cell--header">Details</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card-container</code></td>
+            <td className="nx-cell">Top-Level</td>
+            <td className="nx-cell">
+              Basic card container class.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card-container--row</code></td>
+            <td className="nx-cell">Modifier of <code className="nx-code">.nx-card-container</code></td>
+            <td className="nx-cell">
+              Container for vertically orientated cards (normal and variant). Normal vertical cards can be transformed
+              into horizontal cards by replacing this style with
+              {' '}<code className="nx-code">.nx-card-container--column</code> below.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card-container--column</code></td>
+            <td className="nx-cell">Modifier of <code className="nx-code">.nx-card-container</code></td>
+            <td className="nx-cell">
+              Container for horizontally orientated cards. Horizontal cards can be transformed into vertical cards by
+              replacing this style with <code className="nx-code">.nx-card-container--row</code> above.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              Basic card element.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card-header</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              Card header element, typically wraps an <code className="nx-code">&lt;h3&gt;</code>.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__call-out</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              Wraps the card call out (if any). The call out typically consists of a number, sparkline, icon, or
+              other small graphic. It should not be free form text.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-call__call-out-text</code></td>
```

Comment:
Could this be a modifier of `.nx-card__call-out` instead?

Reply:
> Could this be a modifier of `.nx-card__call-out` instead?

I didn't do it that way because I was trying to avoid the bare text node. `NxCard` is closer to `NxTile` in philosophy than other components in that we're developing something that is a container for other content and while we are demonstrating a variety of options we really don't know what designers will choose to use it for.

Reply:
> I didn't do it that way because I was trying to avoid the bare text node. NxCard is closer to NxTile in philosophy than other components in that we're developing something that is a container for other content and while we are demonstrating a variety of options we really don't know what designers will choose to use it for.

I don't see how our bare text node concerns apply here: the issue we usually worry about with that is when a bare text node is a sibling of an element.  That's not the case here though.  Since you draw a parallel to tiles, I'll say that I feel like this is akin to requiring `.nx-tile-header__subtitle` to have another element underneath it instead of including the subtitle text directly.

Reply:
> > I didn't do it that way because I was trying to avoid the bare text node. NxCard is closer to NxTile in philosophy than other components in that we're developing something that is a container for other content and while we are demonstrating a variety of options we really don't know what designers will choose to use it for.
> 
> I don't see how our bare text node concerns apply here: the issue we usually worry about with that is when a bare text node is a sibling of an element. That's not the case here though. Since you draw a parallel to tiles, I'll say that I feel like this is akin to requiring `.nx-tile-header__subtitle` to have another element underneath it instead of including the subtitle text directly.

Right, what I'm saying is that I can see the very real possibility that something will be added here that will be a sibling of this text node. I'm anticipating that these cards will be used for far more things, far more designs, than @dturner32  could possibly lay out here so the goal is to make them as future proof as possible without going completely crazy.

Reply:
> Right, what I'm saying is that I can see the very real possibility that something will be added here that will be a sibling of this text node. I'm anticipating that these cards will be used for far more things, far more designs, than @dturner32 could possibly lay out here so the goal is to make them as future proof as possible without going completely crazy.

If someone in the future decides on some substructure that fits into this slot it would be up to that substructure to make sure it has no bare text nodes as siblings of elements.  I don't see how requiring this extra element even in cases that aren't that helps anything.

Reply:
> > Right, what I'm saying is that I can see the very real possibility that something will be added here that will be a sibling of this text node. I'm anticipating that these cards will be used for far more things, far more designs, than @dturner32 could possibly lay out here so the goal is to make them as future proof as possible without going completely crazy.
> 
> If someone in the future decides on some substructure that fits into this slot it would be up to that substructure to make sure it has no bare text nodes as siblings of elements. I don't see how requiring this extra element even in cases that aren't that helps anything.

Ok: https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/cd45aea566dbdab783874f540aa041f78f82fdf2